core/src/main/java/org/apache/accumulo/core/file/Prefix.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.file;
+
+public enum Prefix {
```

Comment:
Would something more specific than `Prefix` help with clarity?  Something like `FilePrefix`, `MetaPrefix`  or `MetaFilePrefix`?  Prefix is rather generic and does not help much in usage context for the type / purpose of the prefix.

Reply:
I'm 50/50 on it. If you look at it in the context of it's package, it's pretty clear.

Reply:
It was looking at the other code, for example:

```
TabletFile getNextMapFilename(Prefix prefix) throws IOException 
``
And wondered if there was not an opportunity to add some clarity and context.  Previously, it was just String, so that's not much help either, but with an enum we are declaring a very specific type - adding some additional context would (in my opinion) would help if I was just looking at the other code by pointing to the function / meaning of the prefix without needed to jump to the Prefix definition to see what it really is.  Although, I'd probably need to do that and once I sae "I", "F", - it would be ohh, those prefixes.

Reply:
I have no issue renaming it to FilePrefix. Let's see if there are any other suggestions.

Reply:
I think something more specific would be helpful too. Something like `FilePrefix` for `FileTypePrefix`

Reply:
Renamed from Prefix to FilePrefix in 5efbc1c.